testing/test_HeMIS.py

```diff
@@ -54,7 +54,7 @@ def test_HeMIS(p=0.0001):
             "name": "HeMISUnet",
             "dropout_rate": 0.3,
             "bn_momentum": 0.9,
-            "depth": 2,
+            "depth": 3,
```

Comment:
why changing this behaviour? how is that related to the purpose of the PR?

Reply:
I don't have any errors in test_model.py and errors are in test_HeMIS.py and test_training_time, 
the first error message from  test_HeMIS.py is " weight of size [256, 512, 1, 1], expected input[4, 1024, 6, 6] to have *512* channels, but got *1024* channels instead"

I think that the main problem comes from [here](https://github.com/ivadomed/ivadomed/pull/552#discussion_r536899063) which the number of filters should be doubled but didn't, So, for the HeMIS condition, the number of input channel is twice the [last layer in the encoder] section(https://github.com/AshkanTaghipour/ivadomed/blob/d24e1a8d6fab17b8f62cee8adc5cd34ffbb2f1dd/ivadomed/models.py#L401) --> 2*512=1024
and the depth layer for HeMIS condition considered 2 which is [3 fact](https://github.com/AshkanTaghipour/ivadomed/blob/45011096a54750f1ba073bb10ed7d0bd60f08fb6/testing/test_HeMIS.py#L100)

Reply:
well, I wouldn't change the condition of the test so the test can pass with your changes (ie: your changes should not impact the behaviour of other tests). Or did i miss something @charleygros @andreanne-lemay ?

Reply:
and I have another question, I read the testing folder of the Ivadomed, as an example, for the 'test_training_time' the parameters of the models have been set such as [here](https://github.com/AshkanTaghipour/ivadomed/blob/45011096a54750f1ba073bb10ed7d0bd60f08fb6/testing/test_training_time.py#L86), so when we want to do some changes in test_model.py as I did, and set the model parameters different from [here](https://github.com/AshkanTaghipour/ivadomed/blob/45011096a54750f1ba073bb10ed7d0bd60f08fb6/testing/test_training_time.py#L86) may cause some errors in other tests?
@charleygros @andreanne-lemay